core/deployment/src/main/java/io/quarkus/deployment/builditem/SystemPropertyBuildItem.java

```diff
@@ -1,12 +1,18 @@
 package io.quarkus.deployment.builditem;
 
+import java.util.Comparator;
+
 import io.quarkus.builder.item.MultiBuildItem;
 
 /**
  * Represents a system property that will be set immediately on application startup.
  *
  */
-public final class SystemPropertyBuildItem extends MultiBuildItem {
+public final class SystemPropertyBuildItem extends MultiBuildItem implements Comparable<SystemPropertyBuildItem> {
```

Comment:
These also should not need to be comparable.

Reply:
+1